src/Tempest/Support/src/StringHelper.php

```diff
@@ -5,75 +5,105 @@
 namespace Tempest\Support;
 
 use Countable;
-
-final readonly class StringHelper
+use function ltrim;
+use function preg_quote;
+use function preg_replace;
+use function rtrim;
+use Stringable;
+use function trim;
+
+final readonly class StringHelper implements Stringable
 {
-    public static function title(string $value): string
+    public function __construct(private string $string = '')
+    {
```

Comment:
This is a personal preference, do with it what you will 😄 
```suggestion
    public function __construct(
        private string $string = '',
    ) {
```